tools/spi_nor_config_generator/pull_flashrom_data.py

```diff
@@ -0,0 +1,461 @@
+#!/usr/bin/env python3
+
+import json
+import os
+import re
+import urllib.request
+import pprint
+
+regex_id = re.compile(r'#define\s+([^\s]+)\s+([^\s]+)\s*(/\* (.*) \*/)?$')
+regex_key_replace = re.compile(r'(\s*)\.([a-z_]+)\s+=[\s\n]+([^,{\[]*)', re.MULTILINE)
+regex_enum = re.compile(":\s*([A-Z][A-Z0-9_]*),")
+regex_func = re.compile(":\s*([a-z][a-z0-9_]*),")
+regex_comment = re.compile(r'/\*(.*?)\*/([^\n]*?)$', re.DOTALL|re.MULTILINE)
+regex_model = re.compile("'model_id': ([^,]*),")
+regex_name = re.compile("'name':\\s*\"([^\"]*)\",\\s*?(#[^#]*?)?$", re.MULTILINE)
+
+regex_list = re.compile('{([^:{},]+)([^:{}]*?)}', re.DOTALL)
+regex_list_begin = re.compile('{(\\s*){')
+regex_list_end = re.compile('}([,\\s]*)}')
+
+
+import enum
```

Comment:
This enum should be part of the litespi library (rather than included here).

Reply:
I moved it right [here](https://github.com/litex-hub/litespi/pull/1/files?file-filters%5B%5D=.patch&file-filters%5B%5D=.py&file-filters%5B%5D=No+extension&file-filters%5B%5D=dotfile&hide-deleted-files=true#diff-e07142bb7261ca9dcc8fc3ec66b0ecef)